docs/content/build/5-mainnet.md

```diff
@@ -0,0 +1,41 @@
+# Mainnet
```

Comment:
This page is titled `5-mainnet`, but is 3rd in the sidebar (defined in screenshot below). Let's be consistent – the numbers in the title of the file are for convenience (documentation on how we use this [here](https://github.com/omni-network/omni/tree/main/docs/content), but feel free to update it).

![Screenshot 2024-10-30 at 1 29 57 PM](https://github.com/user-attachments/assets/cede01b4-822a-4e00-9f69-669610f82656)

Reply:
Bothered me too. I started updating the numbers but reverted to make the review easier, without deleted / new files. Suggest another PR to actually remove the numbering on the file names.